variables.tf

```diff
@@ -201,7 +201,7 @@ variable "vacancies_pageviews_refresh_cache_task_command" {
 
 variable "vacancies_pageviews_refresh_cache_task_schedule" {
   description = "vacancies_pageviews_refresh_cache schedule expression - https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html"
-  default     = "cron(0 08 * * ? *)"
+  default     = "cron(0 02 * * ? *)"
```

Comment:
Any reason why this is done at 2am, and not at midnight? I guess it won't matter too much if our daily counts are slightly off for a couple of hours, as I doubt (hope) very much no one is looking at vacancy stats in the wee small hours!

Reply:
We just thought it probably best to run this potentially intensive task at a time when less people are on the service. No real reason for 2am over 12am other than I supposed there will be even less people on at 2am

Reply:
I think 2am is fine in that case